docs/data/base/getting-started/usage/usage.md

```diff
@@ -0,0 +1,117 @@
+# MUI Base - Usage
+
+<p class="description">Learn the basics of working with MUI Base components.</p>
+
+## Getting started
+
+The following code snippet demonstrates a simple app that uses the MUI Base [`ButtonUnstyled`](/base/react-button) component:
+
+```jsx
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import ButtonUnstyled from '@mui/base/ButtonUnstyled';
+
+function App() {
+  return (
+    <div>
+      <ButtonUnstyled>
+        Hello World
+      </ButtonUnstyled>
+    </div>
+  );
+}
+```
+
+You can play around with this code in the interactive Code Sandbox demo below.
+Try importing an `InputUnstyled` component and adding it to the `<div>`:
+
+{{"demo": "Usage.js", "hideToolbar": true, "bg": true}}
+
+## Shared props
+
+Base components are self-supporting and fully functional in isolation.
+Each component has its own unique API, but all components accept the following shared props:
+
+### components
+
+The `components` prop is an object that lets you override any interior subcomponents ("slots") of the base component itself.
+
+> Each component contains a `Root` slot, and many complex components (such as [`BadgeUnstyled`](/base/react-badge)) are composed of additional subcomponents that can be customized.
+
+You can use the `components` prop to replace subcomponents with either custom components or HTML elements.
+
+For example, the [`BadgeUnstyled`](/base/react-badge) component renders a `<span>` by default.
+The code snippet below shows how to override this by assigning a `<div>` to the `Root` element:
+
+```jsx
+<BadgeUnstyled components={{ Root: 'div' }} />
+```
+
+### component
+
+The (singular) `component` prop provides a shortcut to `components.Root`.
+This is useful if you are only overriding the `Root` element of the component.
+
+The code snippet below shows how to override the `Root` element of the [`BadgeUnstyled`](/base/react-badge) component using the `component` prop:
+
+```jsx
+<BadgeUnstyled component="div" />
+```
+
+### componentsProps
+
+The `componentsProps` prop is an object that contains the props for all slots within a component.
+You can use it to define additional custom props for a component's interior elements.
+
+For example, the code snippet below shows how to add a custom CSS class to the `input` slot of the `BadgeUnstyled` component:
+
+```jsx
+<BadgeUnstyled componentsProps={{ input: { className: 'my-badge' } }} />
+```
+
+All additional props placed on the primary component are also propagated into the `Root` slot (just as if they were placed in `componentsProps.root`).
+
+These two examples are equivalent:
+
+```jsx
+<BadgeUnstyled id="badge1">
+```
+
+```jsx
+<BadgeUnstyled componentsProps={{ root: { id: 'badge1' } }}>
+```
+
+## Components vs. hooks
+
+MUI Base includes two kinds of building blocks: **components** and **hooks**.
+
+> 💡 Hooks encapsulate _logic_; components provide _structure_.
+
+Many Base components are implemented with the help of hooks.
```

Comment:
Should we link to the React documentation in case the reader is not 100% familiar with the hooks concept and its intricacies?

Reply:
That's a good idea!